src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -130,6 +138,120 @@ public void cachesGetRequest() throws Exception {
     }
 
     /**
+     * LastModifiedCachingWire can evict any previous cached entry if a new
+     * response does not have a last modified header.
+     * We can observe this via the If-Modified-Since headers as when the cache
+     * does not contain an entry, this is not present on the request.
+     * @throws Exception If fails
+     */
+    @Test
+    public void doesNotCacheGetRequestIfTheLastModifiedHeaderIsMissing()
+        throws Exception {
+        final String first = "Body 1";
+        final String second = "Body 2";
+        final String third = "Body 3";
+        final Map<String, String> noHeaders = Collections.emptyMap();
+        final MkContainer container = new MkGrizzlyContainer()
+            .next(
+                new MkAnswer.Simple(
+                    HttpURLConnection.HTTP_OK,
+                    Collections.singletonMap(
+                            HttpHeaders.LAST_MODIFIED,
+                            "Wed, 15 Nov 1995 05:58:08 GMT"
+                    ).entrySet(),
+                    first.getBytes()
+                ),
+                Matchers.not(queryContainsIfModifiedSinceHeader())
+            )
+            .next(
+                new MkAnswer.Simple(
+                    HttpURLConnection.HTTP_OK,
+                    noHeaders.entrySet(),
+                    second.getBytes()
+                ),
+                queryContainsIfModifiedSinceHeader()
+            )
+            .next(
+                new MkAnswer.Simple(
+                    HttpURLConnection.HTTP_OK,
+                    noHeaders.entrySet(),
+                    third.getBytes()
+                ),
+                Matchers.not(queryContainsIfModifiedSinceHeader())
+            ).start();
+        try {
+            final Request req = new JdkRequest(container.home())
+                .through(LastModifiedCachingWire.class);
+            req.fetch().as(RestResponse.class)
+                .assertStatus(HttpURLConnection.HTTP_OK)
+                .assertBody(
+                    Matchers.equalTo(first)
+            );
+            req.fetch().as(RestResponse.class)
+                .assertStatus(HttpURLConnection.HTTP_OK)
+                .assertBody(
+                    Matchers.equalTo(second)
+            );
+            req.fetch().as(RestResponse.class)
+                .assertStatus(HttpURLConnection.HTTP_OK)
+                .assertBody(
+                    Matchers.equalTo(third)
+            );
+        } finally {
+            container.stop();
+        }
+    }
+
+    /**
+     * LastModifiedCachingWire can resist cache eviction in the event of a non
```

Comment:
Awkward language use because second word must be `can`. I would rather say `LastModifiedCachingWire does not evict cache in the event of a non-OK response`